dev/com.ibm.websphere.appserver.features/visibility/public/acmeCA-2.0/resources/l10n/com.ibm.websphere.appserver.acmeCA-2.0.properties

```diff
@@ -0,0 +1,17 @@
+###############################################################################
+# Copyright (c) 2019 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+#
+#ISMESSAGEFILE FALSE
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+#
+
+description=This feature retrieves a Certificate Authority (CA) signed server certificate by using the ACME protocol during server startup or to renew an expiring certificate.
```

Comment:
is the meaning that this feature obtains the certificate by using ACME _when_ you start the server or renew an expiring certificate or _in order to_ start a server or renew a certificate? Is obtaining the certificate the main benefit of the function or is the certificate just a necessary part of startup/renewal?

also- note that IBM terminology specifies certificate authority is not capitalized

Reply:
On second thought, I think I'm over complicating this for the feature. How about: This feature retrieves a certificate authority (CA) signed server certificate by using the ACME protocol.

Or: This feature retrieves a certificate authority (CA) signed server certificate by using the ACME protocol during server start up or by user request.

Reply:
How about:

```
Feature that implements the Automated Certificate Management Environment (ACME) protocol.
```

Reply:
Or:

```
This feature automates interactions with a certificate authority (CA) using the Automated Certificate Management Environment (ACME) protocol.
```

Reply:
@jvanhill @kristip17 What user goals does the feature primarily support?  Both of your second options are concise and easy to read, just need to figure out how to emphasize the "why" of using the feature (as opposed to the "what")

Reply:
Automates interactions with the certificate authority.

So instead of the user having to request a certificate from a CA and install it manually, this will request a signed certificate from the CA and install it, automatically.

Reply:
It also does some other stuff like revoking and renewing certificates, but for the most part, the concise description is that it automates interactions with the CA.

Reply:
I updated it to Jesse's last suggestion